samples/matter/light_bulb/sample.yaml

```diff
@@ -2,19 +2,36 @@ sample:
   name: Matter Light Bulb
   description: Matter Light Bulb example
 tests:
-  matter.light_bulb:
+  matter.light_bulb.debug:
     build_only: true
     platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp nrf21540dk_nrf52840
```

Comment:
It will not build the target with DFU enabled for nRF21540, but the debug configuration for nRF21540dk_nrf52840 that has DFU off. I believe this part is ok, but there is rather a problem with documentation that says debug option has DFU enabled. I'm not sure yet how this should be solved to avoid misunderstanding.

Reply:
Oh the documentation doesn't say debug has DFU enabled, but mentions no_dfu option what suggests so. i will try to rephrase it somehow.